scripts/package

```diff
@@ -2,7 +2,10 @@
 
 cd $(dirname $0)/..
 
-CONTENT=$(</proc/sys/kernel/random/uuid)
```

Comment:
It's just OS X doesn't have it and it was inconvenient to not be able to run the script just because of that. Also, just noticed, alpine's default shell (`sh`) doesn't support it (`bash` does though). 

`uuidgen` is more portable, so I thought we'd better use it.

If these scripts are only intended to run in dapper containers, I'm fine to revert to `</proc/sys/kernel/random/uuid`.